source/Xunit.Gherkin.Quick.UnitTests/GherkinScenarioOutlineTests.cs

```diff
@@ -307,12 +307,578 @@ public void ApplyExampleRow_Digests_Row_Values_Into_Scenario_With_DocString_In_S
+            "the result for 5 and 6 should be True",          // thenResult1
+            "I declare: \"-5 <= -6\"",                        // givenResult2
+            "the result for -5 and -6 should be False")]      // thenResult2
+        [InlineData(
+            "I declare: \"<a><=<b>\"",                        // givenExpression
+            "the result for <a> and <b> should be <result>",  // thenExpression
+            "I declare: \"5<=6\"",                            // givenResult1
+            "the result for 5 and 6 should be True",          // thenResult1
+            "I declare: \"-5<=-6\"",                          // givenResult2
+            "the result for -5 and -6 should be False")]      // thenResult2
+        [InlineData(
+            "I declare: \"<a> <=<b>\"",                       // givenExpression
+            "the result for <a> and <b> should be <result>",  // thenExpression
+            "I declare: \"5 <=6\"",                           // givenResult1
+            "the result for 5 and 6 should be True",          // thenResult1
+            "I declare: \"-5 <=-6\"",                         // givenResult2
+            "the result for -5 and -6 should be False")]      // thenResult2
+        [InlineData(
+            "I declare: \"<a><= <b>\"",                       // givenExpression
+            "the result for <a> and <b> should be <result>",  // thenExpression
+            "I declare: \"5<= 6\"",                           // givenResult1
+            "the result for 5 and 6 should be True",          // thenResult1
+            "I declare: \"-5<= -6\"",                         // givenResult2
+            "the result for -5 and -6 should be False")]      // thenResult2
+        public void ApplyExampleRow_Digests_Row_Values_Into_Scenario_With_Operators_And_Spaces(
+            string givenExpression,
+            string thenExpression,
+            string givenResult1,
+            string thenResult1,
+            string givenResult2,
+            string thenResult2)
+        {
+            void ValidateStep(Gherkin.Ast.Step step, string keyword, string text, Gherkin.Ast.Step other, bool shouldSubstitute)
+            {
+                Assert.NotSame(other, step);
+                Assert.Equal(shouldSubstitute, other.Text != step.Text);
+
+                Assert.Equal(keyword, step.Keyword);
+                Assert.Equal(text, step.Text);
+            }
+
+            void ValidateScenario(Gherkin.Ast.Scenario scenario, Gherkin.Ast.ScenarioOutline outline, string[] stepTexts)
+            {
+                Assert.NotNull(scenario);
+                Assert.Equal(outline.Name, scenario.Name);
+                Assert.Equal(outline.Steps.Count(), scenario.Steps.Count());
+                Assert.Equal(3, scenario.Steps.Count());
+
+                var sutSteps = outline.Steps.ToList();
+                var scenarioSteps = scenario.Steps.ToList();
```

Comment:
Improve variable names. Please assume this comment for every variable and argument name you have introduced.

Reply:
`scenarioSteps` is something being used before me in other tests.

Reply:
Going to leave `scenario`, `sut`, `scenarioSteps` and `sutSteps` the same for all the old and new tests. But I've fixed all my new names.